website/docs/suggested-projects/typescript.md

```diff
@@ -0,0 +1,20 @@
+---
+title: TypeScript
+slug: /suggested-projects/typescript
+---
+
+## When to test
+
+- Your package runs in projects using TypeScript
```

Comment:
I would update this to read:

```suggestion
- Your package could be run in a project written using TypeScript. Since TypeScript is a superset of JavaScript, most, if not all, JavaScript packages will work in TypeScript projects, so this environment is highly recommended.
```